validator/glb_external_ip.rego

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GLBExternalIpAccessConstraintV1
+
+import data.validator.gcp.lib as lib
+
+###########################
+# Find allowlist Violations
+###########################
+deny[{
+	"msg": message,
+	"details": metadata,
+}] {
+	constraint := input.constraint
+	lib.get_constraint_params(constraint, params)
+	asset := input.asset
+	asset.asset_type == "compute.googleapis.com/GlobalForwardingRule"
+
+	# Find network access config block w/ external IP
+	instance := asset.resource.data
```

Comment:
Maybe `instance` isn't the best term?

Reply:
after discussion instance is still a valid term